keyboards/santoku/keymaps/shaymdev/keymap.c

```diff
@@ -0,0 +1,262 @@
+			_______, 	  KC_PERC, KC_CIRC, KC_LBRC, KC_RBRC, KC_GRV,      _______, KC_P1,   KC_P2,   KC_P3,  KC_PENT, _______, \
+		                               _______, _______, _______,          KC_DEL,  _______, KC_KP_0
+			),
+
+	[_RAISE] = LAYOUT( \
+			_______, KC_PSCR, KC_HOME, KC_UP,   KC_END,  KC_PGUP,     KC_VOLU, KC_F7,   KC_F8,   KC_F9,  KC_F10,  XXXXXXX,\
+			KC_CAPS, KC_INS,  KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN,     KC_VOLD, KC_F4,   KC_F5,   KC_F6,  KC_F11,  XXXXXXX,\
+			_______, KC_NLCK, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     KC_MUTE, KC_F1,   KC_F2,   KC_F3,  KC_F12,  XXXXXXX,\
+			                           _______, _______, _______,     _______, _______, _______
+			),
+
+	[_ADJUST] =  LAYOUT( \
+			TO_DV, RESET,   _______, _______, _______, _______,       _______, _______, _______, _______, _______, KC_DEL,  \
+			RGB_TOG, RGB_MOD, VLK_TOG, AU_ON,   AU_OFF,  AG_NORM,     AG_SWAP,  QWERTY,  DVORAK, _______, _______, _______, \
+			RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,     _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY,  \
+			                           _______, _______, _______,     _______, _______, _______
+			)
+};
+
+
+void keyboard_post_init_user(void) {
+	// Customise these values to desired behaviour
+	debug_enable=false;
+	debug_matrix=false;
+	debug_keyboard=false;
+	debug_mouse=false;
+	//ps2_mouse_set_remote_mode();
+}
+
+const key_override_t delete_key_override = ko_make_basic(MOD_MASK_SHIFT, KC_BSPACE, KC_DELETE);
+
+// This globally defines all key overrides to be used
+const key_override_t **key_overrides = (const key_override_t *[]){
+    &delete_key_override,
+    NULL // Null terminate the array of overrides!
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	switch (keycode) {
+		case RESET:
+			oled_write_ln_P(PSTR("RESETORFLASH"), true);
+			_delay_ms(1500);
+			break;
+
+		case ONETAPALTTAB:
+			// Macro to handle lower-tab as alt-tab
+			if (record->event.pressed) {
+				if (!in_alttab) {
+					SEND_STRING(SS_DOWN(X_LALT) SS_TAP(X_TAB));
+					in_alttab = true;
+				} else {
+					SEND_STRING(SS_TAP(X_TAB));
+					// Do not release Alt here, or it will be impossible to switch more than one window:
+					// alt-tab-tab will be interpreted as alt-tab, then tab
+				}
+			}
+			return false;
+			break;
+
+		default:
+			if ((keycode != ONETAPALTTAB) && in_alttab) {
+				// Exit alt tab before treating normally the keycode
+				SEND_STRING(SS_UP(X_LALT));
+				in_alttab = false;
+				return false;   
+			}
+
+	}
+	return true;
+}
+
+uint16_t get_tapping_term(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case ALT_T(KC_O):
+        case GUI_T(KC_A):
+        case ALT_T(KC_SPC):
+            return TAPPING_TERM + 100;
+        default:
+            return TAPPING_TERM;
+    }
+}
+
+//#ifdef OLED_DRIVER_ENABLE
+#ifdef OLED_ENABLE
```

Comment:
This section should be moved to the keyboard's c file, instead of being here.  That provides a default behavior, which is especially useful for those using the configurator. 

Additionally, once develop is merged into master (at the end of Nov, the function will change to a bool, and `oled_task_kb` will be added.  You don't need to update it now.  But at that point, you will.

Reply:
Cool! Thanks for the heads up.